### ISSUE TITLE
May I ask if this project can be used for multimodal remote sensing image registration? Thanks

### ISSUE THREAD

**Issue by xiaoyu30108** [MIDA-group/CoMIR#60]

**Comment by joakimlindblad** [comment]
Yes that is definitely possible. Actually one of the considered datasets in the paper, the "Zurich Dataset", is QuickBird satellite data (https://zenodo.org/records/5914759).

In case you need only rigid alignment, you may also try https://github.com/MIDA-group/globalign which is a rather different but also very well performing method which does not require any training.

Good luck! (and feel free to reach back to us)
/ Joakim

**Comment by ChangHsinChih** [comment]
Hi, I tried globalign 1.0.3. When I input a pair of **512*512** images (one optical image, one SAR image), it aligns well, but it takes more than 3 seconds. I noticed that the paper says that this situation does not exceed 0.1 seconds. I turned on GPU acceleration and my GPU is an RTX 4060 laptop. How can I solve this problem? Thanks！

![image](https://github.com/user-attachments/assets/69a35d63-5c57-48af-a779-fe5c5615fcfc)
![image](https://github.com/user-attachments/assets/fc09c749-4a9d-429c-be51-9f81f263f558)
